### DISCUSSION TITLE
Second page of the four tiles for most used features

### DISCUSSION THREAD

**Discussion by GalvanicGlaze** [Jyers/Marlin#36] [Ideas]
It would be great to see a second or maybe even third page of the four tiles on the first screen for the most used items like preheat, auto home, load and unload filament, manual bed leveling, z offset, disable steppers. Just to save you for going through the sub menu's. 

I know it doesn't save that much time but I think it would be nice to have

**Comment by Jyers** [comment]
In concept I like the idea, but I think it would end up feeling a little clunky on the display. Switching pages would be a lot less intuitive then a scrolling menu and submenu system. The dwin also only has a few square tiled icons and I can not change the text without flashing new icons on to the screen.

**Reply by Benetos** [reply]
What about under advanced or in place of the leveling button? It could be a "User defined actions" section or something or leveling could be moved under the square next to printing?
